zproject/computed_settings.py

```diff
@@ -384,6 +384,9 @@ def get_dirs(self) -> List[PosixPath]:
     "authenticate_by_username": [
         (1800, 5),  # 5 login attempts within 30 minutes
     ],
+    "create_realm_by_ip": [
+        (3600, 3),
```

Comment:
Thoughts on what limits exactly we should be using here?

Reply:
I would probably use the same value we have for `authenticate_by_username` for these.

Reply:
Makes sense, adjusted it